src/main/java/net/macmv/rgen/block/BambooDoor.java

```diff
@@ -0,0 +1,5 @@
+package net.macmv.rgen.block;
+
+public class BambooDoor extends RgenDoor {
+  // You can add custom properties or methods for the BambooDoor here if needed
```

Comment:
don't organize like this though, its confusing and empty classes are pointless

Reply:
485b8d30ab4a547b61caefdb359946de5f08bb33 resolved